stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/EditText.kt

```diff
@@ -0,0 +1,19 @@
+package io.getstream.chat.android.ui.utils.extensions
+
+import android.app.Activity
+import android.content.Context
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+
+internal fun EditText.hideKeyboard() {
+    val imm: InputMethodManager =
+        context.getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
+    imm.hideSoftInputFromWindow(this.windowToken, 0)
+}
+
+internal fun EditText.showKeyboard() {
+    requestFocus()
+    val imm: InputMethodManager =
+        context.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+    imm.showSoftInput(this, InputMethodManager.SHOW_IMPLICIT)
+}
```

Comment:
This duplicates existing utility methods https://github.com/GetStream/stream-chat-android/blob/1f16150c4025e5dc14ab7203f39d600998e92b55/stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/Utils.java#L119-L145

Reply:
nice catch, thanks!